pkg/template/event.go

```diff
@@ -80,15 +77,20 @@ func newEvent(body []byte, headers http.Header) (*event, error) {
 	}, nil
 }
 
-// applyEventValuesToParams returns a slice of Params with the JSONPath variables replaced
+// ApplyEventValuesToParams returns a slice of Params with the JSONPath variables replaced
```

Comment:
Doesn't look like this is being used anywhere outside of this package. Can we leave this unexported?

Reply:
done!